### DISCUSSION TITLE
How does it work?

### DISCUSSION THREAD

**Discussion by zuldero** [BrianLima/UWPHook#77] [Q&A] [answered]
I tried one time for my self messing with the files of the Xbox Game Pass files. For that i needed to give myself premissions for the trustedinstaller, and it was a big mistake (for me).

Does this App messing with this trustedinstaller? or how do it work?

I dont want to messing around with this trustedinstaller.

Thx for your work and time xD

**Comment by BrianLima** [comment, marked as answer]
Hi @zuldero , this app does not mess with the trusted installer at all, it uses an oficial Windows API to load a list of IDs and names of the UWP apps  and games installed on your computer, via the [Get-AppxPackageManifest powershell ](https://docs.microsoft.com/en-us/powershell/module/appx/get-appxpackagemanifest?view=windowsserver2022-ps)function, contained in our [GetAUMID Script](https://github.com/BrianLima/UWPHook/blob/master/UWPHook/Resources/GetAUMIDScript.ps1), later, by calling the ApplicationActivationManager ActivateApplication C# function using the ID of the requested application, we can launch the application without even touching the permissions system in the computer.

For many people, actually UWPHook was the first way add Windows store games to Steam that actually work, because it does not need to mess with security measures in the system.

If this answer is enough, you may close the thread :D

**Comment by zuldero** [comment]
thats pretty impressive thx for the fast reply and this greate work
